webdriver/tests/classic/new_session/merge.py

```diff
@@ -41,42 +41,57 @@ def test_merge_invalid(new_session, add_browser_capabilities, key, value):
 
 
 def test_merge_platform_name(new_session, add_browser_capabilities, target_platform):
+    always_match = add_browser_capabilities({"timeouts": {"script": 10}})
+
+    if "platformName" in always_match:
+        platform_name = always_match.pop("platformName")
+    else:
+        platform_name = target_platform
+
+    if "pageLoadStrategy" in always_match:
```

Comment:
Similar to `timeouts`. Would you mind to as well move out the `pageLoadStrategy` handling into its own test? It's not needed to test `platformName`.